scripts/sct_register_multimodal.py

```diff
@@ -569,13 +569,20 @@ def main(args=None):
 
 # register images
 # ==========================================================================================
-def register(src, dest, paramreg, param, i_step_str):
+def register(src, dest, paramreg, param, i_step_str, src_seg=None, dest_seg=None):
```

Comment:
we might be able to do without these 2 additional params, if we consider src and dest to accept lists